src/main/java/uk/gov/hmcts/reform/slc/health/PdfServiceHealthIndicator.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.slc.health;
+
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.pdf.service.client.PDFServiceClient;
+
+@Component
+public class PdfServiceHealthIndicator extends AbstractHealthIndicator {
+
+    private final PDFServiceClient client;
+
+    public PdfServiceHealthIndicator(PDFServiceClient client) {
+        this.client = client;
+    }
+
+    @Override
+    protected void doHealthCheck(Health.Builder builder) throws Exception {
+        Health health = client.serviceHealthy();
+        builder.status(health.getStatus());
+        health.getDetails().forEach(builder::withDetail);
```

Comment:
as a bonus (if you just use the interface), you can just returning this health object.

Reply:
(see https://github.com/hmcts/cmc-claim-store/blob/master/src/main/java/uk/gov/hmcts/cmc/claimstore/healthcheck/PDFServiceHealthIndicator.java)